examples/replace_node.yml

```diff
@@ -0,0 +1,11 @@
+---
+- remote_user: root
+  gather_facts: no
+  hosts: server
+  vars:
+    - gluster_maintenance_old_node: host1.example.com
+    - gluster_maintenance_new_node: host1.example.com
+    - gluster_maintenance_cluster_node: data
```

Comment:
minor nit - missed changing cluster_node to be in FQDN format

Reply:
My bad, fixed the FQDN.

I actually found a solution for that. For a given brick directory for example: /foo/bar/baz/bin

I check if any of the part of the path is a mounted filesystem. Like /foo, /foo/bar, /foo/bar/baz ... 

for dir in ${var//\// }; do p=$p/$dir ; if mountpoint -q $p; then echo $p; break; fi ;  done

Now I'm thinking maybe it is a overkill. I will remove that part.